packages/amplify-graphql-relational-transformer/src/__tests__/test-helpers.ts

```diff
@@ -4,11 +4,11 @@ import { FeatureFlagProvider } from '@aws-amplify/graphql-transformer-interfaces
  * Merges passed in feature flags with default feature flags for tests
  */
 export const featureFlags: FeatureFlagProvider = {
-  getBoolean: (value: string): boolean => {
+  getBoolean: (value: string, defaultValue: boolean): boolean => {
     if (value === 'useSubUsernameForDefaultIdentityClaim') {
       return true;
     }
```

Comment:
Can we move the default values to a new hash map and use that here? In that way, if we have to check for any new flags in the future, we don't have to add multiple if statements in the method.

Reply:
We can potentially OOS for this change. I actually added a simple createFeatureFlagProviderWithOverrides util in another PR that can just become our standard to simplify behavior and keep us from reimplementing on a per-test basis. Your call if you think it's blocking or not.

Reply:
I agree with this in theory and am happy to follow up, but since this isn't an established pattern (and there's a mix of using these mock objects in the current tests) is it okay if consolidating the tests is a follow up? If this is blocking I can fix it, but the reason I had to add this feature flag to each test is because this is how the other tests do it.